ruby/ql/lib/codeql/ruby/ApiGraphs.qll

```diff
@@ -268,6 +268,7 @@ module API {
     /**
      * Gets a node representing the `contents` stored on the base object.
      */
+    pragma[inline]
```

Comment:
I wouldn't expect `getAReadContent` to have large fan-in (only large fan-out).

Reply:
As discussed in chat, this just gives a better join order, it's not that it's too large to materialize.